src/store/item-store.js

```diff
@@ -70,6 +70,28 @@ class ItemStore extends Store {
     }
 
 }
+function getItemToEdit(itemAddress){
+    let result = readAddress(itemAddress);
+    if (result.type !== 'item') {
+        throw new Error('Address is not an item.');
+    }
+    return items[result.index];
+}
+function changeTimestamp(itemAddress, date){
+    let itemToChange = getItemToEdit(itemAddress);
+    let timestamp = moment(date);
+    let hour = parseInt(timestamp.format('H'));
+    let minute = parseInt(timestamp.format('m'));
```

Comment:
Yea those do nothing, forgot to delete them from when I copied and pasted. I'll delete them in the next commit.